.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: test jest
+
+on:
+  push: # se ejecuto el action cada que se hace push a las ramas espesificadas, no es necesario a mas
+    branches:
+      - main
+      - develop 
+
+jobs: 
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    
+    - name: Checkout Repository # acceder al codigo del repositorio. hace un checkout del repo y acceder a el para su uso
+      uses: actions/checkout@v4   
+    - name: Setup Node.js # configurar node
+      uses: actions/setup-node@v4
+      with:
+        node-version: "18"
```

Comment:
**chore** 🧹
Create a `.nvmrc` file with the Node version of the project. See: https://github.com/nvm-sh/nvm

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>